src/components/instances/new/MetaLocal.js

```diff
@@ -50,7 +51,23 @@ function MetaLocal() {
 				setFormState({ error: 'usernames must have only letters, underscores, and hyphens' });
 			} else if (instance_name.length && user.length && pass.length && host.length && port.length) {
 				try {
-					const currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
+					let login_not_supported;
+					let currentUser;
+					if (pass) {
+						const login_result = await login({ auth: { user, pass }, url });
```

Comment:
Please be consistent with your casing. It seems much of the code is camelcase, so all new code should be camelcase.